Dev Server Build Failure: Module "support-sponsors.json"

I updated to current master.  When I ran "npm start" to build a local dev server, the documentation site did not load on port 3000.  It's a blank white page.  There is an error in console that states:

Uncaught Error: Cannot find module './support-sponsors.json'.
    at eval (support\-.*\.json$:8)
    at webpackContextResolve (support\-.*\.json$:8)
    at webpackContext (support\-.*\.json$:5)
    at Support.render (support.jsx:45)
    at eval (ReactCompositeComponent.js:796)
    at measureLifeCyclePerf (ReactCompositeComponent.js:75)
    at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext 



Does anyone else have this issue? I ran "npm cache clean", "npm install", then "npm start" to ensure I have the latest packages on my local.

Screenshot:
![webpack-js-fail](https://cloud.githubusercontent.com/assets/1554280/25647696/54217b96-2f92-11e7-8732-041d721155c6.png)

--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   },
   "scripts": {
     "start": "npm run init:generated && npm run fetch && node ./bootstrap.js",
-    "build": "npm run init:generated && rm -rf build/ && node ./bootstrap.js && npm run sitemap && echo webpack.js.org > build/CNAME",
+    "build": "npm run init:generated && npm run fetch && rm -rf build/ && node ./bootstrap.js && npm run sitemap && echo webpack.js.org > build/CNAME",
     "build-test": "npm run build && http-server build/",
     "deploy": "gh-pages -d build",
     "fetch": "scripts/fetch.sh",

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "webpack.js.org",
-  "version": "0.0.0",
+  "version": "0.1.2",
   "private": true,
   "description": "The main site for all things webpack.",
   "homepage": "https://github.com/webpack/webpack.js.org",

--- a/package.json
+++ b/package.json
@@ -24,7 +24,7 @@
     "node": ">=6.9"
   },
   "scripts": {
-    "start": "npm run init:generated && node ./bootstrap.js",
+    "start": "npm run init:generated && npm run fetch && node ./bootstrap.js",
     "build": "npm run init:generated && rm -rf build/ && node ./bootstrap.js && npm run sitemap && echo webpack.js.org > build/CNAME",
     "build-test": "npm run build && http-server build/",
     "deploy": "gh-pages -d build",

--- a/scripts/deploy.sh
+++ b/scripts/deploy.sh
@@ -7,7 +7,6 @@ SOURCE_BRANCH="master"
 # Pull requests and commits to other branches shouldn't try to deploy, just build to verify
 if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "$SOURCE_BRANCH" ]; then
     echo "Skipping deploy; just doing a build and linting links/prose/js."
-    npm run fetch
     npm run build
     npm test
     exit 0
@@ -16,9 +15,6 @@ fi
 # Save some useful information
 REPO=`git config remote.origin.url`
 SSH_REPO=${REPO/https:\/\/github.com\//git@github.com:}
-
-# Fetch loaders/plugins etc.
-npm run fetch
 
 # Run our build
 npm run build

